### ISSUE TITLE
Callback actions in trivial grammar

### ISSUE THREAD

**Issue by jeffreykegler** [jeffreykegler/Marpa--R2#247]
From IRC: http://irclog.perlgeek.de/marpa/2015-02-10#i_10093545
```
#!env perl
use strict;
use diagnostics;
use Marpa::R2;
use Data::Dumper;

print "\$Marpa::R2::VERSION=$Marpa::R2::VERSION\n";

my $s = do { local $/; <DATA>; };
my $g = Marpa::R2::Scanless::G->new({source => \$s});
print Dumper($g->parse(\'', {trace_terminals => 999, trace_values => 999}));

sub nullAction {
  return 'not null';
}

__DATA__
:default ::= action => [values]
lexeme default = action => [start,length,value,name]

:start ::= fake

fake ::=          action => main::nullAction
fake ::= 'unused'
```
```
$Marpa::R2::VERSION=2.103004
Setting trace_terminals option
Setting trace_values option
Expecting "[Lex-0]" at earleme 0
Registering semantics for rule: [:start]
  Semantics are callback
Registering semantics for nulling: fake
  Semantics are callback
Registering semantics for rule: [Lex-0]
  Semantics are push_one 0 result_is_array
Registering semantics for rule: fake
  Semantics are result_is_rhs_n 0
Registering semantics for nulling: [:start]
  Semantics are result_is_rhs_n 0
Registering semantics for token: [:start]
  Semantics are result_is_token_value
Registering semantics for token: [Lex-0]
  Semantics are result_is_token_value
Registering semantics for token: fake
  Semantics are result_is_token_value
value event: starting op MARPA_STEP_NULLING_SYMBOL result_is_rhs_n
$VAR1 = \undef;
```

**Comment by jeffreykegler** [comment]
@jddurand Thanks for finding this.  Want to check it before marking this closed?

**Comment by jddurand** [comment]
I confirm this is fixed with $Marpa::R2::VERSION=2.10300. Thanks.
